setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(
+    name="konf",
+    version="0.1.0",
+    install_requires=[
```

Comment:
I think it will be better to continue using the requirements.txt.

`install_requires=requirements`

Reply:
Is that syntax allowed?

Reply:
Oh no, you will have to read the file before the setup method, something like:

```
with open('requirements.txt') as f:
    requirements = f.read().splitlines()
```

Sorry I saw someone else code but he is doing that before

Reply:
You can ignore me here this is not a big deal if you want to keep it is fine but we should then remove the requirements.txt.

Do you know if the renovate bot works with setup.py dependencies?

Reply:
I think it does, I don't mind reading the requirements file, just wondering if the syntax you had mentioned worked out of the box. Let's just wait on the discussion below before implementing.